packages/common/src/merge.ts

```diff
@@ -1,5 +1,9 @@
-const merge = (target, source) => {
-  const newTarget = { ...target };
+type AnyObject = {
+  [key: string]: any;
+};
```

Comment:
Just a recommendation, but it would be better to avoid the use of `any`, and make use of  `unknown`

```suggestion
type UnknownObject = Record<string, unknown>;
```

if you use change it, you would need to cast in the `merge` function as well

```ts
newTarget[key] = merge(target[key] as UnknownObject || {}, source[key] as UnknownObject);
```

Reply:
I agree! 
Thanks for the suggestion @eduardobreno 🚀

Updated